### PR TITLE
Make third party users join the status room

### PR DESCRIPTION
**Pull request by erdnaxeli**
The goal is that the status room act like a contact list, and in a futur be able to start a new discussion with one or more user.

For this, two main changes:
* now every third party user joining a room will also join the status room
* a new method is provided to give the full contact list and make then join the status room

### CODE REVIEW

File: src/base.js [hunk 1]
```diff
@@ -243,40 +243,20 @@ class Base {
...
   /**
-   * Send a message to the status room
-   *
-   * @param {object} options={} Optional options object: fixedWidthOutput:boolean
-   * @param {string} ...args additional arguments are formatted and send to the room
+   * Async call to get the status room ID
    *
-   * @returns {Promise}
+   * @params {_roomAliasLocalPart} Optional, the room alias local part
+   * @returns {Promise->string} Promise resolving the room ID
```
Comment (kfatehi) [inline src/base.js]: is {Promise->string} actually correct syntax? I think I've done this in the past and it makes jsdoc throw

**Comment by kfatehi** [comment]
Thanks!! So this looks good based on a cursory glance, but I will look a little more in depth later (also providing time in case you are going to push more commits). I'd like to merge it soon just because the diff seems to touch a lot of places... making a merge with some other unrelated PR potentially difficult. This makes me think it may be time to start modularizing some of this stuff.

File: src/base.js (hunk 1 above)
Comment (erdnaxeli) [inline src/base.js]: Hmm, I think I've read it somewhere but I can't find where anymore…

**Comment by erdnaxeli** [comment]
> This makes me think it may be time to start modularizing some of this stuff.

I totally agree :p

File: src/base.js (hunk 1 above)
Comment (kfatehi) [inline src/base.js]: It's discussed at length here https://github.com/jsdoc3/jsdoc/issues/509 but no solution seems to have come of it. Removing for now so docs still gen.

**Pull request merged**